LEAF_Request_Portal/js/formPrint.js

```diff
@@ -340,9 +340,16 @@ var printer = function() {
                                 .text()) : '';
                             textSub();
                             break;
+                        case 'currency':
```

Comment:
I would round the cents up to the nearest 2 decimal points, in cases like $100.555.This would keep it consistent to the normal view which already does this.